pkg/apis/core/v1alpha1/generated.proto

```diff
@@ -516,9 +516,6 @@ message Pod {
   // Tilt uses this to associate Pods with the build that triggered them.
   optional string podTemplateSpecHash = 10;
 
-  // UpdateStartedAt is when Tilt started a deployment update for this Pod.
-  optional k8s.io.apimachinery.pkg.apis.meta.v1.Time updateStartedAt = 11;
```

Comment:
do you know how/if the generator avoids reusing deprecated field ids?

Reply:
Did a quick test - any new fields will get +1 of the highest, so interim deleted fields will be ignored, so we'd need to be careful about deleting the highest field number.

(In general, we should probably avoid deleting fields and use multiple versions + conversions, but since this is brand new and totally wrong, removing it seemed to make more sense - nothing external should be relying on the value of this field right now.)